dds_cli/cli_decorators.py

```diff
@@ -191,43 +191,35 @@ def create_and_remove_task(self, *args, **kwargs):
 
             # Determine spinner text
             if func.__name__ == "remove_all":
-                description = f"Removing all files in project {self.project}..."
+                description = f"Removing all files in project {self.project}"
             elif func.__name__ == "remove_file":
-                description = "Removing file(s)..."
+                description = "Removing file(s)"
             elif func.__name__ == "remove_folder":
-                description = "Removing folder(s)..."
+                description = "Removing folder(s)"
 
             # Add progress task
-            task = progress.add_task(description=description)
+            task = progress.add_task(description=f"{description}...")
 
-            # Execute function
-            message = func(self, *args, **kwargs)
```

Comment:
Is the progress task removed in this case? So that the spinner is not displayed if it fails?

Reply:
Tried it out now and it doesn't, when the message was caught and then the `progress.remove_task` below was used, the spinner line was removed. 
With `message = func(self, *args, **kwargs)` and `progress.remove_task(task)`:
```
Failed to delete files in project: {
    "message": "The project is empty.",
    "status": 400
}
```

With `func(self, *args, **kwargs)`:
```
Removing all files in project project_1... ⠀⠠
ERROR    Failed to delete files in project: {                                                                                                                      
             "message": "The project is empty.",                                                                                                                   
             "status": 400                                                                                                                                         
         }                                                                                                                                                         
             
```